isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -280,6 +280,75 @@ OpenID Connect Abfragen zur Identität eines Benutzers, OpenID-Token oder Custom
 
 [[authentifizierung-und-autorisierung]]
 === Authentifizierung & Autorisierung
+Gemäß der xref:isyfact-standards-doku:referenzarchitektur:fachlich.adoc#[IsyFact-Referenzarchitektur] wird bei der Authentifizierung nach vordefinierten xref:isyfact-standards-doku:referenzarchitektur:software-technisch.adoc#prinzipien-der-sicherheitsarchitektur[Prinzipien der Sicherheitsarchitektur] unterschieden, wie das nachfolgende Bild zeigt:
+
+.Authentifizierungswege als Mandanten
+[id="image-authorization-code-connections",reftext="{figure-caption} {counter:figures}"]
+image::konzept/security_multi-tenancy_with_multi_iams.dn.svg[align="center"]
+
+Dabei ergeben sich die unterschiedlichen Kommunikationskanäle _Portal_, _Service-Gateway_, _Authentication-Gateway_ oder _interne Anwendungs-Authentifizierung_ (Batch/Task/Kontextwechsel), über welche ein Authentifizierungs-Request an der Anwendungslandschaft ankommt bzw. verarbeitet wird.
```

Comment:
Die Multi-Realm/Tenancy Thematik ist eine Vorgabe von isy-iam. D.h. (noch) keine Vorgabe der IsyFact-Standards. Daher würde ich hier schreiben, dass aufgrund der unterschiedlichen Zugriffswegen eine Unterscheidung nach Mandanten möglich sein kann und wenn dies gewünscht ist, unser Baustein dafür eine entsprechende Unterstützung bietet. Wichtig ist auch zu nennen, dass jeder Mandant in jeder Anwendung, die für diesen Mandanten ausgestellte Access-Tokens akzeptieren soll, konfiguriert werden muss. Die genaue Konfiguration befindet sich dann in den Nutzungsvorgaben.

Und am besten wir benutzen den Begriff "Tenant" bzw. "Multi-Tenancy", dann ist es einheitlich mit SB/Materna.